mlflow/langchain/runnables.py

```diff
@@ -44,11 +44,15 @@
 
 
 def _load_model_from_config(path, model_config):
-    from langchain.chains.loading import load_chain
     from langchain.chains.loading import type_to_loader_dict as chains_type_to_loader_dict
-    from langchain.llms.loading import get_type_to_cls_dict as llms_get_type_to_cls_dict
-    from langchain.llms.loading import load_llm
-    from langchain.prompts.loading import load_prompt
+
+    try:
+        from langchain.llms.loading import get_type_to_cls_dict as llms_get_type_to_cls_dict
+    except ImportError:
+        try:
+            from langchain_community.llms import get_type_to_cls_dict as llms_get_type_to_cls_dict
```

Comment:
This is new on langchain 0.0.349. Would it be better using `try except` or `Version(...) >= Version("0.0.349")` ?

Reply:
`Version(...) >= Version("0.0.349")` is better.

Reply:
What is langchain_community?

Reply:
I see https://python.langchain.com/docs/get_started/installation#langchain-community